cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6f08ac68",
+   "metadata": {},
+   "source": [
+    "# Itteratively Find Improved Consensus Labels for Multiannotator Data using Cleanlab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44a0555b",
+   "metadata": {},
+   "source": [
+    "This example shows how to improve consensus labels by combining the CROWDLAB algorithm with itteratively retraining a model. For an introductory tutorial on finding consensus labels with the multiannotator library see [Find Best Consensus Labels for Multiannotator Data using Cleanlab](linke to hui wen's). \n",
+    "\n",
+    "The following code uses the [cifar10h](linke cifar10h) multiannotator labeling dataset which is a collection of 2751 annotators each labeling 200 examples for all 10,000 test images of the original [cifar10](link cifar10) dataset but **any multiannotator classification image dataset should work with the code below**. "
```

Comment:
As our multi-annotator dataset, we use a subset of [cifar10h](link cifar10h), which is ...

Reply:
"but **any multiannotator classification image dataset should work with the code below**"
->
"Because this notebook utilizes AutoML for model training and Cleanlab is compatible with any model/dataset, you should be able to run the below code with **any** image classification dataset where each image has been labeled by multiple annotators.